docker-compose.yaml

```diff
@@ -1,10 +1,22 @@
 version: '3.9'
+
+x-defaults:
+  - yugabyte_image: &yb-image ${YB_IMAGE:-yugabytedb/yugabyte:2.15.3.0-b231}
+  - configs: &common
+      networks:
+        - tlr-network
+      extra_hosts:
+        - "host.docker.internal:host-gateway"
```

Comment:
For some reason, my container cannot resolve `host.docker.internal`, but I found this workaround to work for me. I'm not sure if it works for Docker Desktop for MacOS though. If not, we can just comment it out and leave a comment for people with similar issue in the future.

Reply:
I'm reading that Mac and Windows machines are fine without the "extra_hosts" flag, so maybe this should be commented out. Alternatively, we could create a separate docker-compose file for Linux.

Reply:
OK, I've reverted `docker-compose.yaml` to the original file, plus a minor syntax fix, and moved the changes to a `docker-compose-linux.yaml` file instead. Also, updated `local_deployment.md`.